addons/hr_timesheet_invoice/hr_timesheet_invoice.py

```diff
@@ -46,9 +46,8 @@ def _invoiced_calc(self, cr, uid, ids, name, arg, context=None):
         res = {}
 
         cr.execute('SELECT account_id as account_id, l.invoice_id '
-                'FROM hr_analytic_timesheet h LEFT JOIN account_analytic_line l '
-                    'ON (h.line_id=l.id) '
-                    'WHERE l.account_id = ANY(%s)', (ids,))
+                'FROM account_analytic_line l '
```

Comment:
I bet the whole method could be simplified using a read_group on account_analytic_line, grouping on account_id, with a domain is_timesheet, and summing amount_untaxed.

While we are at it, could you add type='float' on amount_invoiced field ?